tokio-threadpool/src/pool/mod.rs

```diff
@@ -45,12 +45,6 @@ pub(crate) struct Pool {
     // Stack tracking sleeping workers.
     sleep_stack: CachePadded<worker::Stack>,
 
-    // Number of workers that haven't reached the final state of shutdown
-    //
-    // This is only used to know when to single `shutdown_task` once the
-    // shutdown process has completed.
-    pub num_workers: AtomicUsize,
-
```

Comment:
This is just an unused field from the times before `ShutdownTrigger` so I'm removing it.